tutorials/iot-device-to-device/index.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Google Cloud IoT Core Device to Device Communication
+description: Learn how to relay messages from one device to another using Google Cloud Functions.
+author: gguuss
+tags: Google Cloud IoT Core, Cloud IoT, Google Cloud Functions
+date_published: 2017-12-05
+---
+
+Sometimes you want to have one device control another device. For example, lets
+say that you have a device (Device 1) that is a switch that reconfigures a
+second device (Device 2) to be on.  In this scenario, you would want for a
```

Comment:
"to be on" seems a little ambiguous. Is this a standard way to talk about devices in IoT? If not, please clarify (e.g. "a switch that turns on a second device (Device 2)").